### PR TITLE
hotspot & name convention update

### PR DESCRIPTION
**Pull request by DannyS03**
https://github.com/OpenLiberty/guides-common/issues/352
https://github.com/OpenLiberty/guides-common/issues/308

### CODE REVIEW

**Comment by DannyS03** [comment]
Arrowhead issue to show code-pane should be fixed when pushed to master

File: README.adoc [hunk 1]
```diff
@@ -174,30 +174,30 @@ Lastly,
 
 [role="edit_command_text"]
 Add a [hotspot=1-5 file=1]`<parent/>` section just above `basic configuration` to each of the `pom.xml` files in the child modules,
```
Comment (ankitagrawa) [inline README.adoc]: image

Comment (ankitagrawa) [reply README.adoc]: Parent hotspot tag needs correction. It's still written in the old way

File: README.adoc [hunk 2]
```diff
@@ -177,7 +177,7 @@ Add a [hotspot=1-5 file=1]`<parent/>` section just above `basic configuration` t
 such as the [hotspot=parent-jar file=2]`jar`, [hotspot=parent-war file=3]`war`, and [hotspot=parent-ear file=4]`ear` projects.
 
 pom.xml
-[source, xml, linenums, role='code_column hide_tags=parent']
+[source, xml, linenums, role='code_column tags=Parent']
```
Comment (ankitagrawa) [inline README.adoc]: image

Comment (ankitagrawa) [reply README.adoc]: I still see parent with small p. Also the hotspot is not working.

Comment (DannyS03) [reply README.adoc]: Yes still trying to figure it out.

**Comment by ankitagrawa** [comment]
image

**Comment by ankitagrawa** [comment]
image

Parent hotspot not working correctly.

**Comment by evelinec** [comment]
All hotspots are updated to use tags in #54, along with the changes to simplified the pom. I think this PR is not needed anymore.

**Comment by DannyS03** [comment]
- applied in dev mode changes: https://github.com/OpenLiberty/guide-maven-multimodules/pull/55

**Pull request closed without merging**